BuildMonitor/Domain/ISolutionBuild.cs

```diff
@@ -6,6 +6,8 @@ public interface ISolutionBuild : ITimer, IPersistable
     {
         DateTime Started { get; }
         ISolution Solution { get;  }
+        bool IsRebuildAll { get; set; } // bit of a shame to make this mutable
```

Comment:
If it's a shame then why make it mutable? ;-) I don't have a big problem with making stuff mutable, but am not a big fan of comments that don't really add any value.